### ISSUE TITLE
Error when launching headless UMS

### ISSUE THREAD

**Issue by Krakinou** [UniversalMediaServer/UniversalMediaServer#3767]
Hello
Release : 13.0.1
I'm trying to use UMS as a headless server on a raspberry Pi on a 64bit Debian (uname -m = aarch64).
When I launch UMS.sh (as root) I got following error message:
````
root@myserver:/opt/ums # ./UMS.sh headless
Exception in thread "main" java.lang.ExceptionInInitializerError
	at net.pms.util.FilePermissions.checkPermissions(FilePermissions.java:208)
	at net.pms.util.FilePermissions.isExecutable(FilePermissions.java:297)
	at net.pms.platform.linux.LinuxProgramPaths.<init>(LinuxProgramPaths.java:65)
	at net.pms.platform.PlatformProgramPaths.<clinit>(PlatformProgramPaths.java:195)
	at net.pms.configuration.UmsConfiguration.<init>(UmsConfiguration.java:745)
	at net.pms.configuration.UmsConfiguration.<init>(UmsConfiguration.java:696)
	at net.pms.PMS.main(PMS.java:955)
Caused by: com.vdurmont.semver4j.SemverException: The build cannot be empty.
	at com.vdurmont.semver4j.Semver.<init>(Semver.java:100)
	at com.vdurmont.semver4j.Semver.<init>(Semver.java:20)
	at net.pms.platform.PlatformUtils.createOsVersion(PlatformUtils.java:369)
	at net.pms.platform.PlatformUtils.<clinit>(PlatformUtils.java:68)
	... 7 more
````
I double checked and I don't see which file permissions may be missing (moreover... I'm logged as root)
I previously had exactly the same settings on a 32bit Debian and didn't have this issue.

**Comment by SubJunk** [comment]
@Krakinou it looks like the operating system version isn't recognized for some reason. I'm looking at it now

**Comment by SubJunk** [comment]
@Krakinou if I look at the code for that third-party semver library, it looks like your operating system is failing the version check because it has a `+` character with nothing after it, see this line https://github.com/vdurmont/semver4j/blob/master/src/main/java/com/vdurmont/semver4j/Semver.java#L100

What does the operating system report its version as?

**Comment by ik666** [comment]
It's probably the same issue I had on the mac ... see #3162

**Comment by Krakinou** [comment]
Is that what you need?
````
 sudo uname -a
Linux myserver.ovh 5.15.76-v8+ #1597 SMP PREEMPT Fri Nov 4 12:16:41 GMT 2022 aarch64 GNU/Linux
````

**Comment by SubJunk** [comment]
Ah yeah, this version `5.15.76-v8+` isn't a valid version under semver, I think. We can do a fix like @ik666 mentioned, I'll upload it for you to test when it's done

**Comment by fybx** [comment]
+1, I'm having the same issue as well

**Comment by SubJunk** [comment]
Can you please see if this fixes it? https://www.universalmediaserver.com/uploads/UMS-3767.tgz

**Comment by Krakinou** [comment]
It worked! Thanks
I assume released in 13.2.0?

**Comment by SubJunk** [comment]
@Krakinou thanks for confirming! The fix didn't make it into 13.2.0 but will be in the next one
